tests/test_build_html.py

```diff
@@ -11,11 +11,12 @@
 
```

Comment:
Review is **NOT** needed.
Converting yield testing into pytest.mark.parametrize